examples/xgboost_model_fitting_adult.ipynb

```diff
@@ -0,0 +1,773 @@
+{
```

Comment:
<p>example example --&gt; example</p><p>"where a classification into two income brackets" --&gt; need to fix sentence</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I'm completely blind.